src/protagonist/API.Tests/Integration/ModifyAssetTests.cs

```diff
@@ -108,6 +108,38 @@ public async Task Put_NewImageAsset_Creates_Asset()
         asset.BatchAssets.Should().BeEmpty();
     }
     
+    [Fact]
+    public async Task Put_NewImageAsset_CreatesAsset_WithManifestsAttached()
+    {
+        var customerAndSpace = await CreateCustomerAndSpace();
```

Comment:
I think this is unnecessary. You can use the name of this method to ensure it's unique. `var assetId = AssetIdGenerator.GetAssetId()` should be enough?